test/internalTest.js

```diff
@@ -632,17 +632,34 @@ for (const supportedVersion of mineflayer.testedVersions) {
             })
           }
 
-          client.write('named_entity_spawn', {
-            entityId: 56,
-            playerUUID: '1-2-3-4',
-            x: 1,
-            y: 2,
-            z: 3,
-            yaw: 0,
-            pitch: 0,
-            currentItem: -1,
-            metadata: []
-          })
+          if (bot.registry.version['>=']('1.20.2')) {
```

Comment:
would be good to make that a feature

Reply:
done, looks like removing dupe https://github.com/PrismarineJS/minecraft-data/commit/587a1341c83ef4fc1cf3353d35b09435a11f8217 caused range to be incorrect, just fixed that